Alon-FX-5

alongolo opened the pull request:
New features in GUI.

idoEfron committed:
Merge branch 'AlonFX' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/AR.fxml

idoEfron committed:
Merge branch 'AlonFX' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/AR.fxml

idoEfron committed:
Merge branch 'master-iteration2' of https://github.com/yinonbz/FootballProject into idoFx

alongolo committed:
Java Fx - Create Season

alongolo committed:
Merge branch 'master-iteration2' of https://github.com/yinonbz/FootballProject into Alon-FX-4

 Conflicts:
	src/main/java/serviceLayer/LeagueService.java
	target/classes/serviceLayer/LeagueService.class

ItaiKatz1 committed:
Added support to online/offline users in the database and their notifications

alongolo committed:
Notification Design Prototype - Fan.fxml

AR - Implemented add referees to season and confirm season's Match Policy

ItaiKatz1 committed:
Added support to online/offline users in the database and their notifications

alongolo committed:
Merge branch 'itai-iteration3' of https://github.com/yinonbz/FootballProject into Alon-FX-4

 Conflicts:
	src/main/java/businessLayer/userTypes/SystemController.java
	src/main/java/businessLayer/userTypes/viewers/Fan.java

alongolo committed:
Notification Design Prototype - Fan.fxml

AR - Implemented add referees to season and confirm season's Match Policy

alongolo committed:
Notification Design Prototype - Fan.fxml

AR - Implemented add referees to season and confirm season's Match Policy

idoEfron committed:
Merge branch 'AlonFX' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/AR.fxml

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/RefereeController.java
#	target/classes/businessLayer/Tournament/Match/MatchController.class
#	target/classes/fxml/AR.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/RefereeController.class

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/RefereeController.java
#	target/classes/businessLayer/Tournament/Match/MatchController.class
#	target/classes/fxml/AR.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/RefereeController.class

alongolo committed:
Notification Design Prototype - Fan.fxml

AR - Implemented add referees to season and confirm season's Match Policy

alongolo committed:
Notification Design Prototype - Fan.fxml

AR - Implemented add referees to season and confirm season's Match Policy

alongolo committed:
Merge branch 'idoFx' of https://github.com/yinonbz/FootballProject into Alon-FX-4

alongolo committed:
1. Fixed some bugs in the AR.
2. Added some exceptions. Tests were updated accordingly.
3. Fixed on-login notifications.
4. New Design (CSS updated).

alongolo committed:
1. Fixed some bugs in the AR.
2. Added some exceptions. Tests were updated accordingly.
3. Fixed on-login notifications.
4. Team Owner - Add Team Manager.
5. New Design (CSS updated).

alongolo committed:
1. Fixed some bugs in the AR.
2. Added some exceptions. Tests were updated accordingly.
3. Fixed on-login notifications.
4. Team Owner - Add Team Manager.
5. New Design (CSS updated).

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/RefereeController.java
#	target/classes/businessLayer/Tournament/Match/MatchController.class
#	target/classes/fxml/AR.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/RefereeController.class

alongolo committed:
Merge branch 'idoFx' of https://github.com/yinonbz/FootballProject into Alon-FX-4

 Conflicts:
	resources/fxml/Player.fxml

alongolo committed:
Merged with Ido

alongolo committed:
Merged with Ido

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/RefereeController.java
#	target/classes/businessLayer/Tournament/Match/MatchController.class
#	target/classes/fxml/AR.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/RefereeController.class

alongolo committed:
Merged with Ido

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/RefereeController.java
#	target/classes/businessLayer/Tournament/Match/MatchController.class
#	target/classes/fxml/AR.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/RefereeController.class

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

alongolo committed:
Merged with Ido

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

alongolo committed:
Merged with Ido

idoEfron committed:
Merge branch 'Alon-FX-5' of https://github.com/yinonbz/FootballProject into ido-FX-5

alongolo committed:
1. Close and Reopen Teams in TeamOwner.fxml
2. Offline notifications seems to be working.

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

alongolo committed:
1. Close and Reopen Teams in TeamOwner.fxml
2. Offline notifications seems to be working.

alongolo committed:
Merge branch 'ido-FX-5' of https://github.com/yinonbz/FootballProject into Alon-FX-5

 Conflicts:
	src/main/java/serviceLayer/SystemService.java
	target/classes/businessLayer/userTypes/SystemController.class
	target/classes/dataLayer/DataBaseValues2.class
	target/classes/serviceLayer/SystemService.class

alongolo committed:
1. Close and Reopen Teams in TeamOwner.fxml
2. Offline notifications seems to be working.

tomersein commented on src/main/java/businessLayer/userTypes/SystemController.java:
```diff
@@ -20,7 +20,6 @@
 import businessLayer.userTypes.Administration.*;
 import businessLayer.userTypes.viewers.*;
 import dataLayer.DemoDB;
```
Not approved.
Every function that you have written in this class you need to put in a comment.
@alongolo @idoEfron

tomersein reviewed (changes requested):
we will talk about the changes that are needed here.
we can't approve it until we know for sure that the business layer doesn't have a duplicate code.

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

idoEfron committed:
Merge branch 'Alon-FX-4' of https://github.com/yinonbz/FootballProject into idoFx

# Conflicts:
#	resources/fxml/AR.fxml
#	resources/fxml/Admin.fxml
#	resources/fxml/Coach.fxml
#	resources/fxml/Player.fxml
#	resources/fxml/Referee.fxml
#	resources/fxml/TeamManager.fxml
#	resources/fxml/TeamOwner.fxml
#	src/main/java/presentationLayer/ARController.java
#	src/main/java/presentationLayer/CoachController.java
#	src/main/java/presentationLayer/FanController.java
#	src/main/java/presentationLayer/LoginController.java
#	src/main/java/presentationLayer/PlayerController.java
#	src/main/java/presentationLayer/RefereeController.java
#	src/main/java/presentationLayer/TeamManagerController.java
#	src/main/java/presentationLayer/TeamOwnerController.java
#	target/classes/fxml/Admin.fxml
#	target/classes/fxml/Coach.fxml
#	target/classes/fxml/Player.fxml
#	target/classes/fxml/Referee.fxml
#	target/classes/fxml/TeamManager.fxml
#	target/classes/fxml/TeamOwner.fxml
#	target/classes/presentationLayer/ARController.class
#	target/classes/presentationLayer/CoachController.class
#	target/classes/presentationLayer/FanController.class
#	target/classes/presentationLayer/LoginController.class
#	target/classes/presentationLayer/PlayerController.class
#	target/classes/presentationLayer/RefereeController.class
#	target/classes/presentationLayer/TeamManagerController.class
#	target/classes/presentationLayer/TeamOwnerController.class

alongolo committed:
All of the functions in the SystemController.java which was using the DB and was used by the presentation layer was inserted into a comment (/* */)

alongolo committed:
All of the functions in the SystemController.java which was using the DB and was used by the presentation layer was inserted into a comment (/* */)
Because of this, many tests are failing. The failing tests were also placed in comments, under a:  ""//todo: check with new DB" comment. Later we will test the tests with the new DB.

tomersein reviewed (approved):
approved.
deleted the old DB, next step is to merge the data layer
